geonode_mapstore_client/client/package.json

```
@@ -17,6 +17,7 @@
   },
   "scripts": {
     "compile": "mapstore-project compile && node postCompile",
+    "compiledev": "MAPSTORE_COMPILE_DEV=true mapstore-project compile && node postCompile",
```

@marlowp I'm just proposing to use similar structure we have for `test:watch` command where we are using `:` char
```suggestion
    "compile:dev": "MAPSTORE_COMPILE_DEV=true mapstore-project compile && node postCompile",
```